Source/Rows/Common/FieldRow.swift

```diff
@@ -268,28 +287,19 @@ open class _FieldCell<T> : Cell<T>, UITextFieldDelegate, TextFieldCell where T:
             if let titleLabel = titleLabel, let text = titleLabel.text, !text.isEmpty {
                 views["label"] = titleLabel
                 dynamicConstraints += NSLayoutConstraint.constraints(withVisualFormat: "H:[imageView]-(15)-[label]-[textField]-|", options: NSLayoutFormatOptions(), metrics: nil, views: views)
-                dynamicConstraints.append(NSLayoutConstraint(item: textField,
-                                                             attribute: .width,
-                                                             relatedBy: (row as? FieldRowConformance)?.textFieldPercentage != nil ? .equal : .greaterThanOrEqual,
-                                                             toItem: contentView,
-                                                             attribute: .width,
-                                                             multiplier: (row as? FieldRowConformance)?.textFieldPercentage ?? 0.3,
-                                                             constant: 0.0))
-            } else {
+                dynamicConstraints.append(NSLayoutConstraint(item: titleLabel, attribute: .width, relatedBy: calculatedTitlePercentage != nil ? .equal : .greaterThanOrEqual, toItem: contentView, attribute: .width, multiplier: calculatedTitlePercentage ?? 0.3, constant: 0.0))
```

Comment:
Since this constraint is applied to `titleLabel` instead of `textField`, the inverse value (i.e. 0.3) should be correct IMHO.
However, your comment did point me to a flaw in my solution when it comes to handling nil values. See [this commit](https://github.com/xmartlabs/Eureka/pull/965/commits/0a8c1a1a52df3fa204c63702f49bf340357edcb9) for more details.